src/testing/utils.tsx

```diff
@@ -151,3 +155,45 @@ export const renderWrappedHook = <Result, Props>(
   });
   return { router, result, store };
 };
+
+export function createStore(
+  state: RootState,
+  opts: { trackActions: true },
+): [EnhancedStore<RootState>, UnknownAction[]];
+export function createStore(
+  state: RootState,
+  opts?: { trackActions?: boolean },
+): EnhancedStore<RootState>;
```

Comment:
I think this return type would need to be `EnhancedStore<RootState> | [EnhancedStore<RootState>, UnknownAction[]]` or else change opts to `opts?: { trackActions: false },`